themes/icons/filter.svg

```diff
@@ -1,3 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
-  <polygon class="stroke-linejoin-round" points="15 2 1 2 6 8 6 15 10 15 10 8 15 2"/>
-</svg>
\ No newline at end of file
+<svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
let's remove `fill=none` for consistency with all other icons

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
```

The rendered UI will not change, because we correct this style anyway: https://github.com/cloudscape-design/components/blob/70bafc50860ae991853c407766ca4d4a11199965/src/icon/mixins.scss#L97-L98

Reply:
Next time, I will not add `fill=none`, thank you for the review!